cli/commands/init/ai_provider_step.py

```diff
@@ -39,38 +39,55 @@ def ai_provider_step(options, default_options, none_interactive, abort):
         default_options["llm_model_name"],
         none_interactive,
     )
-
-    ask_if_not_provided(
+    # Ask if the user wants to enable the embedding service
```

Comment:
Adding the ability to disable the embedding service in the init step.

Alternatively, we could leave always omit the embedding service and make the user manually add it if they wanted to.